proto/crescent/liquidity/v1beta1/query.proto

```diff
@@ -88,6 +89,14 @@ service Query {
   rpc NumMMOrders(QueryNumMMOrdersRequest) returns (QueryNumMMOrdersResponse) {
     option (google.api.http).get = "/crescent/liquidity/v1beta1/num_mm_orders/{orderer}/{pair_id}";
   }
+
+  rpc Trades(QueryTradesRequest) returns (QueryTradesResponse) {
+    option (google.api.http).get = "/crescent/liquidity/v1beta1/trades";
+  }
+
+  rpc Trade(QueryTradeRequest) returns (QueryTradeResponse) {
+    option (google.api.http).get = "/crescent/liquidity/v1beta1/trades/{id}";
+  }
```

Comment:
I think would be nice if there was a comments for swagger generation or description field that said this endpoint is enabled by the node flag

Reply:
I'll add comments.